src/src/models/SelectModel.ts

```diff
@@ -0,0 +1,18 @@
+import OptionModel from './OptionModel';
+
+export default interface SelectInputModel {
+    propName: String;
+    error: String;
+    disabled: Boolean;
+    mandatory: Boolean;
+    value: String;
+    id: String;
+    tabindex: number;
+    customClasses: String;
+    hasValidation: Boolean;
+    isValid: Boolean;
+    showLabel: Boolean;
+    showDefault: Boolean;
+    defaultLabel: String;
+    options: OptionModel[];
+};
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
};;
```